### ISSUE TITLE
Build 20.08 on Windows 10 failing without clear error

### ISSUE THREAD

**Issue by zahmbie** [PixarAnimationStudios/USD#1328]
### Description of Issue
Hello! 

I'm trying to compile USD 20.08 on Windows 10 and it's failing but I'm not finding any errors in the log files to point me in the direction of what's failing.  The build process ends with the following message:

ERROR: Failed to run 'cmake --build . --config Release --target install -- /M:16'
See D:\USD\Pixar\builds_test\build\Pixar\log.txt for more details.

I'm at a bit of a loss on where to start investigating from here.  Any help pointing me towards what the failure is would be appreciated!

### Steps to Reproduce
1. Use x64 Native Tools Command Prompt for VS 2017 as administrator to run the following:
python build_scripts\build_usd.py --force-all --verbose "D:\USD\Pixar\builds_test"

### System Information (OS, Hardware)
Windows 10
Visual Studio Community 2017
Python x64 v3.7.9

[CMakeError.log](https://github.com/PixarAnimationStudios/USD/files/5234309/CMakeError.log)
[CMakeOutput.log](https://github.com/PixarAnimationStudios/USD/files/5234310/CMakeOutput.log)
[log.txt](https://github.com/PixarAnimationStudios/USD/files/5234311/log.txt)

**Comment by sunyab** [comment]
Hi @zahmbie, from the log.txt you posted I see:
```
CUSTOMBUILD : error : D:\USD\Pixar\pxr\usdImaging\usdviewq\mainWindowUI.ui: Warning: The name 'attributeBrowserFrame' (QFrame) is already in use, defaulting to 'attributeBrowserFrame1'. [D:\USD\Pixar\builds_test\build\Pixar\pxr\usdImaging\usdviewq\_usdviewq_pysideuifiles.vcxproj]
```
This looks like the issue with the newest version of PySide2 that was fixed in the dev branch by #1320. So you could either build from the dev branch or install an older version of PySide2 (for example, by doing `pip install PySide2==5.14.1`)

**Comment by zahmbie** [comment]
Ahhh, excellent!  That did it for me, thank you!
